src/Navigation/StackNavigation.js

```diff
@@ -96,6 +97,16 @@ function DrawerRoutes() {
           drawerIcon: profileIcon,
         }}
       />
+      <Drawer.Screen
+        name="TermDrawer"
+        component={TermsScreen}
+        options={{
+          title: 'Terms',
+          drawerIcon: profileIcon,
+          headerTitle: '',
+          headerTransparent: true,
+        }}
+      />
```

Comment:
Is this supposed to be here? I don't see it on the figma, so I assume this was supposed to be removed